tests/refl.py

```diff
@@ -213,4 +223,53 @@ def test_GIVEN_component_in_beam_WHEN_starting_up_ioc_THEN_inbeam_sp_true_and_po
     def test_GIVEN_motor_values_set_WHEN_starting_refl_ioc_THEN_parameter_rbvs_are_initialised_correctly(self):
         expected = IN_COMP_INIT_POS
 
-        self.ca.assert_that_pv_is("PARAM:IN_POS", expected)
\ No newline at end of file
+        self.ca.assert_that_pv_is("PARAM:IN_POS", expected)
+
+    def test_GIVEN_motor_velocity_altered_by_move_WHEN_move_completed_THEN_velocity_reverted_to_original_value(self):
+        expected = INITIAL_VELOCITY
+        self.set_up_velocity_tests(expected)
+
+        self.ca.set_pv_value("PARAM:THETA:SP", 22.5)
+
+        self.ca_galil.assert_that_pv_is("MTR0102.DMOV", 1, timeout=10)
+        self.ca_galil.assert_that_pv_is("MTR0102.VELO", expected)
+        self.ca_galil.assert_that_pv_is("MTR0103.DMOV", 1, timeout=10)
+        self.ca_galil.assert_that_pv_is("MTR0103.VELO", expected)
+
+    def test_GIVEN_motor_velocity_altered_by_move_WHEN_move_interrupted_THEN_velocity_reverted_to_original_value(self):
+        expected = INITIAL_VELOCITY
+        self.set_up_velocity_tests(expected)
+
+        # move and wait for completion
+        self.ca.set_pv_value("PARAM:THETA:SP", 22.5)
+        self.ca_galil.set_pv_value("MTR0102.STOP", 1)
+        self.ca_galil.set_pv_value("MTR0103.STOP", 1)
+        self.ca_galil.set_pv_value("MTR0104.STOP", 1)
+
+        self.ca_galil.assert_that_pv_is("MTR0102.DMOV", 1, timeout=2)
```

Comment:
Can you do a check here to see if the move has actually been interrupted? It may just be that the move has finished naturally.